sdk/go/common/workspace/plugins.go

```diff
@@ -1530,11 +1575,17 @@ func getPluginInfoAndPath(
 		logging.V(6).Infof("GetPluginPath(%s, %s, %s): enabling new plugin behavior", kind, name, version)
 		candidate, err := SelectCompatiblePlugin(plugins, kind, name, semver.MustParseRange(version.String()))
 		if err != nil {
-			return nil, "", NewMissingError(PluginInfo{
-				Name:    name,
-				Kind:    kind,
-				Version: version,
-			}, includeAmbient)
+			// could not find a compatible plugin
```

Comment:
This only handles the case where we try and lookup a plugin with a version. If we just try to look up "aws" for example we'll hit the codepath below which never attempts this, it _could_ because plugins mostly support looking up their latest version, and that's probably a sensible one to pick so I wonder if we should re-jig to support that?

Reply:
Actually the path below with missing plugin error is intercepted in GetPluginInfo which _can_ auto install a plugin without a version (for example with `pulumi convert`) so I think this case covered

Reply:
> So basically removing the missing plugin error altogether?

Well no we might still get a missing plugin. Either the download could fail, or if we have no version get_latest_version could fail.

> is intercepted in GetPluginInfo

Yes I think get rid of that and just move it to here instead.

Reply:
Done ✔️